app/src/androidTest/java/com/github/gogetters/letsgo/utils/PermissionUtilsTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.github.gogetters.letsgo.utils
+
+import android.Manifest
+import android.content.pm.PackageManager
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.gogetters.letsgo.utils.PermissionUtils.requestPermission
+import com.github.gogetters.letsgo.utils.PermissionUtils.isPermissionGranted
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.Assert.*
+
+@RunWith(AndroidJUnit4::class)
+class PermissionUtilsTest {
+
+    @Test
+    fun requestPermissionTest() {
+        // TODO implement
+        assertTrue(true)
+    }
+
+    @Test
+    fun isPermissionGrantedWorksWithoutPermission() {
```

Comment:
I would change the name to make it more evident that the test should reject (e.g.) rejectIfNoPermission